hooks/pre_build_centos

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash -ex
+yum install docker distgen -y
```

Comment:
we have distgen in Fedora, is epel enabled on your builders?

Reply:
we need to add it, thats a fresh machine and epel is not enabled by default in centos. any thing else we should add as dependency?